cypress/integration/8.Login/login_spec.js

```diff
@@ -1,3 +1,9 @@
+Cypress.on('uncaught:exception', (err, runnable) => {
```

Comment:
I'm a little concerned about ignoring all uncaught exceptions. Is there a particular error that is being thrown?

Reply:
Yes the error thrown is for an `unhandled rejection` . It directs me to this error: https://reactjs.org/docs/error-decoder.html/?invariant=31&args[]=TypeError%3A%20NetworkError%20when%20attempting%20to%20fetch%20resource.&args[]=

I've looked to see where that may be an issue and all I found was a bug with the default image. But after fixing that I still get that error. Console is clear and everything works as its supposed to.

Reply:
Ugh. That's frustrating. Looking at this again...will this only impact the login_spec.js test? Or all Cypress tests in the suite?

Reply:
@sdreher It will only impact the login_spec.js test. But this Ignore pattern is littered throughout the cypress tests so I am going to go ahead and clean those up as well.

Reply:
Cool - I think that's a really good approach, then we can put it back where needed in a more targeted way.